_includes/configuration/media_poster.md

```diff
@@ -0,0 +1,22 @@
+### `media_poster`
+
+This options allows you disable the `Poster` input field in the media dialog.
```

Comment:
This has some grammar issues and isn't even a proper sentence. I think it's missing `to`?

Reply:
```suggestion
To remove the `Poster` input field in the media dialog, set this option to `false`.
```